v22.1/restore.md

```diff
@@ -340,6 +340,52 @@ RESTORE DATABASE bank FROM LATEST IN 's3://{bucket_name}?AWS_ACCESS_KEY_ID={key_
+
+This is different to a backup with revision history that captures every change made within the garbage collection period leading up to and including the given timestamp. See [Take Backups with Revision History and Restore from a Point-in-time](take-backups-with-revision-history-and-restore-from-a-point-in-time.html) for more detail.
+
+In the case where there is no backed up revision history, use the following example to restore to a particular time.
+
+First, find the times that are available for a point-in-time-restore by listing the available backup directories in your storage location:
+
+{% include_cached copy-clipboard.html %}
+~~~sql
+SHOW BACKUPS IN 's3://{bucket_name}?AWS_ACCESS_KEY_ID={key_id}&AWS_SECRET_ACCESS_KEY={access_key}';
+~~~
+~~~
+          path
+------------------------
+  2023/01/18-141753.98
+  2023/01/23-184816.10
+  2023/01/23-185448.11
+(3 rows)
+~~~
+
+From the output use the required date directory and run the following to get the details of the backup:
+
+~~~sql
+SHOW BACKUP '2023/01/23-185448.11' IN 's3://{bucket_name}?AWS_ACCESS_KEY_ID={key_id}&AWS_SECRET_ACCESS_KEY={access_key}';
+~~~
+~~~
+  database_name | parent_schema_name |        object_name         | object_type | backup_type |         start_time         |          end_time          | size_bytes | rows | is_full_cluster
+----------------+--------------------+----------------------------+-------------+-------------+----------------------------+----------------------------+------------+------+------------------
+  movr          | public             | vehicle_location_histories | table       | full        | NULL                       | 2023-01-23 18:54:48.116975 |      85430 | 1092 |        t
+  movr          | public             | promo_codes                | table       | full        | NULL                       | 2023-01-23 18:54:48.116975 |     225775 | 1003 |        t
+  movr          | public             | user_promo_codes           | table       | full        | NULL                       | 2023-01-23 18:54:48.116975 |       1009 |   11 |        t
+  NULL          | NULL               | system                     | database    | incremental | 2023-01-23 18:54:48.116975 | 2023-01-24 00:00:00        |       NULL | NULL |        t
+  system        | public             | users                      | table       | incremental | 2023-01-23 18:54:48.116975 | 2023-01-24 00:00:00        |          0 |    0 |        t
+  system        | public             | zones                      | table       | incremental | 2023-01-23 18:54:48.116975 | 2023-01-24 00:00:00        |          0 |    0 |
+~~~
+
+Use the `start_time` and `end_time` detail to define the required time as part of the `AS OF SYSTEM TIME` clause. Run the restore, passing the directory and the timestamp:
```

Comment:
consider: `Lastly,` or `Finally,` to follow neatly from `First` above.

If you went with my `path` earlier, this would need to change also. If not: not!